packages/core/src/modules/credentials/repository/CredentialExchangeRecord.ts

```diff
@@ -46,8 +46,6 @@ export interface CredentialRecordBinding {
   credentialRecordType: CredentialFormatType
   credentialRecordId: string
   credentialId?: string
-  indyRevocationRegistryId?: string
-  indyCredentialRevocationId?: string
```

Comment:
`indyRevocationRegistryId ` and `indyCredentialRevocationId ` were originally in the `DefaultCredentialTags`, and I think this got mixed up with the merge from main. Can you move them back?